Mathlib/Topology/Algebra/InfiniteSum/Defs.lean

```diff
@@ -108,6 +133,13 @@ noncomputable irreducible_def tprod {β} (f : β → α) :=
     else h.choose
   else 1
 
+variable {L : SummationFilter β}
+
+@[inherit_doc tprod]
+notation3 "∏'[" L "]" (...)", "r:67:(scoped f => tprod f L) => r
+@[inherit_doc tsum]
+notation3 "∑'[" L "]" (...)", "r:67:(scoped f => tsum f L) => r
+
 -- see Note [operator precedence of big operators]
 @[inherit_doc tprod]
 notation3 "∏' "(...)", "r:67:(scoped f => tprod f) => r
```

Comment:
maybe add the `unconditional` in the notation definition, to spare some work to Lean, and also to make sure that `(∏' b, f b ) unconditional` (or whatever inserting `unconditional` at the right place to confuse Lean) fails.